app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/fragments/DatasetListRecyclerViewAdapter.kt

```diff
@@ -24,6 +25,7 @@ class DatasetListRecyclerViewAdapter(
      */
     private val NB_REPRESENTATIVES_SHOWN = 3
 
+    // TODO Use injection with Hilt!
     private val dbMgt: DatabaseManagement = DummyDatabaseManagement.staticDummyDatabaseManagement
```

Comment:
in the meantime, you can use `DummyDatabaseManagement(DummyDatabaseService())` instead of static: safer.

Reply:
yup, you're right